src/ess/diffraction/uncertainty.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+"""Tools for handling statistical uncertainties."""
+
+from .types import RawData, RawDataWithvariances, VanadiumRun
+
+
+def drop_variances(data: RawDataWithvariances[VanadiumRun]) -> RawData[VanadiumRun]:
+    res = data.copy(deep=False)
+    if res.bins is not None:
+        res.bins.constituents['data'].variances = None
+    else:
+        res.variances = None
+    return RawData[VanadiumRun](res)
+
+
+providers = (drop_variances,)
+"""Sciline providers for handling statistical uncertainties."""
```

Comment:
Why are the vanadium errors dropped?

Reply:
Because we normalise in event mode and that requires broadcasting a vanadium histogram to sample events.

Reply:
Maybe we should not do that?